packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/login/index.astro

```diff
@@ -39,34 +40,30 @@ if (locals.isLoggedIn) {
       </div>
     <br />
 
-      <!-- oAUTH Providers -->
       <OauthProviders dashboardConfig={dashboardConfig} />
-      <!-- End of oAUTH Providers -->
 
-      <!-- Username and Password Provider-->
       { usernameAndPassword && (
         <div class="py-8">
           <div class="bg-base-300 p-8 rounded-lg md:w-[32rem]">
 
           <form method="post" action={loginAPI}>
 
-              <sl-input 
+              <Input 
                 name="username" 
                 id="username" 
-                >
-                  <div class="text-xl font-bold" slot="label">Username</div>
-              </sl-input><br />
+                label='Username'
+                /><br />
 
-              <sl-input 
+              <Input 
                 type="password" 
                 name="password"
                 id="password"
-                password-toggle>
-                  <div class="text-xl font-bold" slot="label">Password</div>
-                </sl-input><br />
+                label="Password"
+                password-toggle
+                /><br />
 
             <center>
```

Comment:
center!

Reply:
This was not a CSS/Layout update, this was a Shoelace.style update